backend/app/core/venue_service.py

```
@@ -1,101 +1,122 @@
-from sqlalchemy.orm import Session
-from sqlalchemy import and_, or_, func, text, extract
-from typing import List, Optional, Dict, Any, Tuple
-from datetime import datetime, date, timedelta
-from decimal import Decimal
-import uuid
 import logging
-from math import radians, cos, sin, asin, sqrt
+import uuid
+from datetime import date, datetime, timedelta
+from decimal import Decimal
+from math import asin, cos, radians, sin, sqrt
```

Ruff (0.11.9)

`math.asin` imported but unused

Remove unused import

(F401)